### ISSUE TITLE
No icons on map

### ISSUE THREAD

**Issue by dceejay** [darkplusplus/opentakrouter#34]
hi I'm sending COT like
```
<event version="2.0" type="a-h-g-u-c-i-z" uid="205th Separate Motorized Rifle Brigade" time="2022-04-03T16:09:17.840Z" start="2022-04-03T16:09:17.840Z" stale="2022-04-03T16:19:17.840Z" how="h-e"><point lat="46.890757" lon="33.144856" hae="9999999" le="9999999.0" ce="9999999.0"/><detail><takv device="DCJmac.lan" os="darwin" platform="NRedTAK" version="0.5.4"/><track course="0" speed="0"/><contact callsign="205th Separate Motorized Rifle Brigade"/><remarks source="NRedTAK"> #Hostile</remarks></detail></event>
```
and the console show things like
```
17:09:23 INF] server=tak-tcp endpoint=192.168.1.193:57384 session=7aed3207-d380-4f72-8a98-e37862fdc74d event=cot uid=205th Separate Motorized Rifle Brigade type=a-h-g-u-c-i-z
```
so it seems to be received - but nothing appears on the map ui

It is entirely possible my CoT is messed up - but not sure how (works with Taky and FTS) - any clues ?

**Comment by gregbuehler** [comment]
I modified the time values on the event and chucked it at the public instance and I'm seeing the relevant event being received and rendered. This may be an issue with either propagating the event to the web client or it may be a library issue. 

If you can help investigate that'd be awesome! From the map view, if you look in the inspector, can you verify that you see the following elements in the console?

- The milsymbol library loading
- The web socket connection announcing it is connected
- The received event(s)

If everything is working as expected, we'd expect it to look like this:
![image](https://user-images.githubusercontent.com/1753943/161441411-c0b801ef-2e6b-420c-894e-80e8f193f42a.png)

**Comment by gregbuehler** [comment]
This may be resolved by #37 and #38. It seems jquery was not properly bundled in v1.0.12 and as a result the map events were not being handled correctly for fresh installs.

**Comment by dceejay** [comment]
Yes - 1.0.14 fixed it for me... Thanks
